<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open_folder"/>
    <addaction name="separator"/>
    <addaction name="action_save_mask"/>
    <addaction name="action_mask_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_save_img"/>
    <addaction name="action_img_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_last"/>
    <addaction name="action_next"/>
    <addaction name="separator"/>
    <addaction name="action_run"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="action_tree"/>
    <addaction name="action_config"/>
    <addaction name="action_list"/>
    <addaction name="separator"/>
    <addaction name="action_in"/>
    <addaction name="action_out"/>
    <addaction name="action_origin"/>
    <addaction name="separator"/>
    <addaction name="action_fit_window"/>
    <addaction name="action_fit_width"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_help"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_tree">
   <property name="windowTitle">
    <string>目录结构</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_folder"/>
   <addaction name="separator"/>
   <addaction name="action_save_mask"/>
   <addaction name="action_save_img"/>
   <addaction name="separator"/>
   <addaction name="action_last"/>
   <addaction name="action_next"/>
   <addaction name="separator"/>
   <addaction name="action_run"/>
   <addaction name="separator"/>
   <addaction name="action_in"/>
   <addaction name="action_out"/>
   <addaction name="action_origin"/>
   <addaction name="separator"/>
   <addaction name="action_fit_window"/>
   <addaction name="action_fit_width"/>
   <addaction name="separator"/>
   <addaction name="action_close"/>
   <addaction name="action_quit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_config">
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>400</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>项目配置</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>64</height>
        </size>
       </property>
       <property name="title">
        <string>模型选择</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>512</width>
         <height>128</height>
        </size>
       </property>
       <property name="title">
        <string>原始图像</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_hide_img">
          <property name="text">
           <string>是否隐藏</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>不透明度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_img">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_img">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="maximumSize">
        <size>
         <width>512</width>
         <height>128</height>
        </size>
       </property>
       <property name="title">
        <string>掩码图像</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkBox_hide_mask">
          <property name="text">
           <string>是否隐藏</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>不透明度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_mask">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_mask">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_reverse_mask">
          <property name="text">
           <string>掩码反转</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_list">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>137</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>图片文件列表</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QListView" name="listView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_open_folder">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/folder.png</normaloff>:/icon/folder.png</iconset>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save_mask">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/save-mask.png</normaloff>:/icon/save-mask.png</iconset>
   </property>
   <property name="text">
    <string>保存掩码</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_img">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/save-img.png</normaloff>:/icon/save-img.png</iconset>
   </property>
   <property name="text">
    <string>保存图像</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_mask_save_as">
   <property name="text">
    <string>掩码另存为</string>
   </property>
  </action>
  <action name="action_img_save_as">
   <property name="text">
    <string>图像另存为</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/close.png</normaloff>:/icon/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/quit.png</normaloff>:/icon/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_tree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>目录结构</string>
   </property>
  </action>
  <action name="action_config">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>项目配置</string>
   </property>
  </action>
  <action name="action_list">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>文件列表</string>
   </property>
  </action>
  <action name="action_in">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/zoom-in.png</normaloff>:/icon/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_out">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/zoom-out.png</normaloff>:/icon/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_origin">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/origin.png</normaloff>:/icon/origin.png</iconset>
   </property>
   <property name="text">
    <string>原始大小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+R</string>
   </property>
  </action>
  <action name="action_fit_window">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/window.png</normaloff>:/icon/window.png</iconset>
   </property>
   <property name="text">
    <string>适应窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="action_fit_width">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/width.png</normaloff>:/icon/width.png</iconset>
   </property>
   <property name="text">
    <string>适应宽度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+H</string>
   </property>
  </action>
  <action name="action_last">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/last.png</normaloff>:/icon/last.png</iconset>
   </property>
   <property name="text">
    <string>上一张图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="action_next">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/next.png</normaloff>:/icon/next.png</iconset>
   </property>
   <property name="text">
    <string>下一张图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="action_run">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/run.png</normaloff>:/icon/run.png</iconset>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
   <property name="toolTip">
    <string>执行</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/help.png</normaloff>:/icon/help.png</iconset>
   </property>
   <property name="text">
    <string>使用方法</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
